public/app/features/manage-dashboards/types.ts

```diff
@@ -2,17 +2,6 @@ import { Dashboard } from '@grafana/schema/src/veneer/dashboard.types';
 
 import { ExternalDashboard } from '../dashboard/components/DashExportModal/DashboardExporter';
 
-export interface Snapshot {
-  created: string;
-  expires: string;
```

Comment:
The values for created/expires/updated are all ignored -- so i removed them from the copy in: https://github.com/grafana/grafana/pull/77763/files#diff-9b1ae42589c4484c3841cc89bedb40095e97ffdc97cd86b05ab9aa1d16b15c4fR5